README.md

```diff
@@ -19,6 +19,14 @@ This tool is intended to help you setup a Dockerrun.aws.json which allows you to
 ### Prerequisites 
 You need to have your AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables set in order to use the tool as well as the proper aws permissions for Elastic Beanstalk, and S3 access. 
 
+Note : From gantree 0.6.16, The necessity / configuration of AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables is not neccessary if gantree is installed in EC2 instance with proper IAM Role and policy.
```

Comment:
Readability / Grammar :

Note : For gantree versions >= 0.6.16, configuration of AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables is not necessary if running on an instance with IAM Roles enabled.